spectrum/test_article.py

```diff
@@ -182,6 +188,7 @@ def _wait_for_published(article):
     checks.ARCHIVE.of(id=article.id(), version=article.version())
     article_from_api = checks.API.article(id=article.id(), version=article.version())
     checks.JOURNAL.article(id=article.id(), volume=article_from_api['volume'], has_figures=article.has_figures())
+    checks.JOURNAL_CDN.article(id=article.id(), volume=article_from_api['volume'], has_figures=article.has_figures())
```

Comment:
additional check that every article that gets published is also accessible through the CDN